### PR TITLE
suite-nonsplit-health-checks

### PR DESCRIPTION
**Pull request by himanshu-saxena-metacube**
https://spryker.atlassian.net/browse/CC-16815 - suite-nonsplit-health-checks

### CODE REVIEW

File: tests/api/suite_nonsplit/glue/utility_endpoints/health_checks/negative.robot [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+### Test works only if all services are disabled as default
+    When I send a GET request:    /health-check
+    Then Response status code should be:    403
+    And Response reason should be:    Forbidden
+    And Response body parameter should be:    [data][0][type]    health-check
+    And Response body parameter should be:    [data][0][id]    None
+    And Response body parameter should be:    [data][0][attributes][status]    None
+    And Response body parameter should be:    [data][0][attributes][statusCode]    403
+    And Response body parameter should be:
+    ...    [data][0][attributes][message]
```
Comment (Katerynaa-Lavryk) [inline tests/api/suite_nonsplit/glue/utility_endpoints/health_checks/negative.robot:28]: Please fix the assertion to have it displayed in one line

File: tests/api/suite_nonsplit/glue/utility_endpoints/health_checks/negative.robot [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+    ...    HealthCheck endpoints are disabled for all applications.
+    And Response should contain the array of a certain size:
```
Comment (Katerynaa-Lavryk) [inline tests/api/suite_nonsplit/glue/utility_endpoints/health_checks/negative.robot:30]: Please fix the assertion to have it displayed in one line

File: tests/api/suite_nonsplit/glue/utility_endpoints/health_checks/negative.robot [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+    When I send a GET request:    /health-check?services=sear
+    Then Response status code should be:    400
+    And Response reason should be:    Bad Request
+    And Response body parameter should be:    [data][0][type]    health-check
+    And Response body parameter should be:    [data][0][id]    None
+    And Response body parameter should be:    [data][0][attributes][status]    None
+    And Response body parameter should be:    [data][0][attributes][statusCode]    400
+    And Response body parameter should be:    [data][0][attributes][message]    Requested services not found.
+    And Response should contain the array of a certain size:
+    ...    [data][0][attributes][healthCheckServiceResponses]
```
Comment (Katerynaa-Lavryk) [inline tests/api/suite_nonsplit/glue/utility_endpoints/health_checks/negative.robot:45]: Please fix the assertion to have it displayed in one line

File: tests/api/suite_nonsplit/glue/utility_endpoints/health_checks/negative.robot [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+    When I send a GET request:    /health-check?services=
+    Then Response status code should be:    400
+    And Response reason should be:    Bad Request
+    And Response body parameter should be:    [data][0][type]    health-check
+    And Response body parameter should be:    [data][0][id]    None
+    And Response body parameter should be:    [data][0][attributes][status]    None
+    And Response body parameter should be:    [data][0][attributes][statusCode]    400
+    And Response body parameter should be:    [data][0][attributes][message]    Requested services not found.
+    And Response should contain the array of a certain size:
+    ...    [data][0][attributes][healthCheckServiceResponses]
```
Comment (Katerynaa-Lavryk) [inline tests/api/suite_nonsplit/glue/utility_endpoints/health_checks/negative.robot:59]: Please fix the assertion to have it displayed in one line

File: tests/api/suite_nonsplit/glue/utility_endpoints/health_checks/negative.robot (hunk 1 above)
Comment (himanshu-saxena-metacube) [reply tests/api/suite_nonsplit/glue/utility_endpoints/health_checks/negative.robot]: Fixed

File: tests/api/suite_nonsplit/glue/utility_endpoints/health_checks/negative.robot (hunk 2 above)
Comment (himanshu-saxena-metacube) [reply tests/api/suite_nonsplit/glue/utility_endpoints/health_checks/negative.robot]: Fixed

File: tests/api/suite_nonsplit/glue/utility_endpoints/health_checks/negative.robot (hunk 3 above)
Comment (himanshu-saxena-metacube) [reply tests/api/suite_nonsplit/glue/utility_endpoints/health_checks/negative.robot]: Fixed

**Review by Katerynaa-Lavryk — APPROVED** [review]

**Pull request merged**